zend_abstract_interface/interceptor/php8/resolver.c

```diff
@@ -5,8 +5,28 @@ static void zai_interceptor_function_declared(zend_op_array *op_array, zend_stri
     zai_hook_resolve_function((zend_function *)op_array, name);
 }
 
+#if PHP_VERSION_ID < 80300
```

Comment:
If this is to work around a bug on two minor releases, why isn't this restricted to just those versions? This patch will also apply to 8.2.2, yes?

Reply:
It will. I wasn't sure whether a version check here is worth it, but probably I should just do it.

Reply:
Done :-)